benchmarks/benchmark_loop/benchmark_definitions.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class BenchmarkDefinition:
```

Comment:
Do you intend this to be limited to blackbox benchmarks? If so, we should call this something like BlackboxBenchmarkDefinition.

Otherwise, we can attempt to unify the meta-data specification for all our benchmarks. This can be done later, but we want to avoid to have to specify this 2x.

Reply:
It should be any list the user wants (possibly real/non simulated benchmark), I think it makes to have a concise standalone definition but I agree we can discuss to see whether there are some parts that would benefit from being refactored.